internal/quickstart/messages.go

```diff
@@ -189,7 +200,21 @@ func fetchEnv(
 		}
 		err = json.Unmarshal(response, &resp)
 		if err != nil {
-			return errMsg{err: err}
+			msgRequestErr, err := newMsgRequestError(err.Error())
+			if err != nil {
+				return errMsg{err: err}
+			}
+
+			if !msgRequestErr.IsConflict() {
```

Comment:
I don't think we can get a conflict error when fetching the environment, so we're probably fine to just return the error every time.